showcase/app/templates/layouts/flex.hbs

```diff
@@ -0,0 +1,495 @@
+    </SF.Item>
+    <SF.Item
+      @label="column"
+      {{style width="25%"}}
+      class="shw-layout-flex-example-outline-flex-container shw-layout-flex-example-tint-flex-items"
+    >
+      <Hds::Layout::Flex @direction="column">
+        <Shw::Placeholder @text="item #1" @height="40" />
+        <Shw::Placeholder @text="item #2" @height="40" />
+        <Shw::Placeholder @text="item #3" @height="40" />
+        <Shw::Placeholder @text="item #4" @height="40" />
+      </Hds::Layout::Flex>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H2>Wrap</Shw::Text::H2>
+
+  <Shw::Flex @direction="column" as |SF|>
+    {{#let (array false true) as |booleans|}}
+      {{#each booleans as |wrap|}}
+        <SF.Item
+          @label={{if wrap "wrap" "nowrap (default)"}}
+          class="shw-layout-flex-example-outline-flex-container shw-layout-flex-example-tint-flex-items"
+        >
+          <div {{style width="650px" max-width="100%"}}>
+            <Hds::Layout::Flex @wrap={{wrap}}>
+              <Shw::Placeholder @text="#1" @width="200" @height="40" {{style flex-shrink="0"}} />
+              <Shw::Placeholder @text="#2" @width="200" @height="40" {{style flex-shrink="0"}} />
+              <Shw::Placeholder @text="#3" @width="200" @height="40" {{style flex-shrink="0"}} />
+              <Shw::Placeholder @text="#4" @width="200" @height="40" {{style flex-shrink="0"}} />
+              <Shw::Placeholder @text="#5" @width="200" @height="40" {{style flex-shrink="0"}} />
+            </Hds::Layout::Flex>
+          </div>
+        </SF.Item>
+      {{/each}}
+    {{/let}}
+  </Shw::Flex>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H2>Justify + Align</Shw::Text::H2>
+  <Shw::Text::Body>These are the
+    <code>justify-content</code>
+    and
+    <code>align-items</code>
+    CSS properties of
+    <code>flexbox</code>.</Shw::Text::Body>
+
+  {{#each @model.DIRECTIONS as |direction|}}
+    <Shw::Text::H3>direction={{direction}}</Shw::Text::H3>
+    <Shw::Grid @columns={{@model.JUSTIFYS.length}} @gap="2rem" as |SG|>
+      {{#each @model.ALIGNS as |align ac|}}
+        {{#if (eq ac 0)}}
+          {{#each @model.JUSTIFYS as |justify|}}
+            <SG.Item>
+              <span class="shw-label">justify={{justify}}</span>
+            </SG.Item>
+          {{/each}}
+        {{/if}}
+        {{#each @model.JUSTIFYS as |justify jc|}}
+          {{! Notice: we're  using an invisible character here to preserve the alignment of the items }}
+          <SG.Item
+            @label={{if (eq jc 0) (concat "align=" align) "​"}}
+            class="shw-layout-flex-example-outline-flex-container shw-layout-flex-example-tint-flex-items"
+          >
+            <div {{style width="120px" height="120px"}}>
+              <Hds::Layout::Flex
+                @direction={{direction}}
+                @justify={{justify}}
+                @align={{align}}
+                {{style width="100%" height="100%"}}
```

Comment:
@KristinLBradley this is one of those cases I mentioned (but there are others in this file) where having `@width` and `@height` arguments directly on the `Flex` component would come in handy

Reply:
I think that width and height properties could be helpful on layout components like these.

(I don't think that we would need to add them to other HDS components as you mentioned since the other components either aren't focused around layout or have more standardized dimensions.)